src/metabase/util/ssh.clj

```diff
@@ -6,12 +6,20 @@
 
 (def ^:private default-ssh-timeout 30000)
 
+(defn ssh-add-key
+  "Adds SSH private key to Jsch class."
+  [jsch key-file-name]
+  (log/debug "Adding ssh key:" key-file-name)
+  (when-not (string/blank? key-file-name) (doto ^com.jcraft.jsch.JSch jsch (.addIdentity key-file-name)))
```

Comment:
I don't think `str` is a namespace. And `string/blank?` will also check for strings containing only whitespace. Might save headaches down the line if a user errantly enters a blank space in the field.

See https://clojuredocs.org/clojure.core/empty_q and https://clojuredocs.org/clojure.string